packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -71,6 +71,20 @@ export const styles = (theme) => ({
       transform: 'translateX(100%)',
     },
   },
+  /* Styles applied when the component is passed children. */
+  withChildren: {
+    '& > *': {
+      visibility: 'hidden',
+    },
+  },
+  /* Styles applied when the component is passed children and no width. */
+  fitContent: {
+    maxWidth: 'fit-content',
+  },
+  /* Styles applied when the component is passed children and no height. */
+  heightAuto: {
+    height: 'auto',
+  },
```

Comment:
I noticed it is still handy to be able to set `width` when passing children, and then figured the same would be useful with the `height` prop.